lib/travis/build/addons/phantomjs.rb

```diff
@@ -0,0 +1,72 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class PhantomJs < Base
+        SUPER_USER_SAFE = true
+
+        attr_reader :version
+
+        def after_prepare
+          sh.fold 'install_phantomjs' do
+            sanitize(raw_version)
+
+            unless version
+              sh.echo "Invalid version '#{raw_version}' given. Only 1.9.8 and 2.0.0 are supported at the moment", ansi: :red
+              return
+            end
+
+            sh.echo "Installing PhantomJS #{version}", ansi: :yellow
+
+            sh.if "$(uname) = 'Linux'" do
+              sh.if "$(lsb_release -c) = 'precise'" do
```

Comment:
The 2.0.0 binary exists for Trusty. https://s3.amazonaws.com/travis-phantomjs/phantomjs-2.0.0-ubuntu-14.04.tar.bz2